### PR TITLE
Certificate 101

### PR DESCRIPTION
**Pull request by shubh0779**

### CODE REVIEW

**Comment by shubh0779** [comment]
Add Certificate file

**Review by laxmikant-gaykhee — CHANGES_REQUESTED** [review]

File: SubmittedAssignments/Shubham_Shinde/Preview.html [hunk 1]
```diff
@@ -0,0 +1,92 @@
...
+			<!-- <div class="tri6"></div> -->
+			<!-- <div class="boxbtm"></div> -->
+			<!-- <div class="boxtop"></div> -->
+		<!-- </div> -->
+
+		<!--Content of Certificate-->
+		<div class="certificate-content">
+			<div class="title">Certificate</div>
+			<div class="sr-no">Certificate Number: 1234</div>
+			<div class="Date">Date: 14-07-2024</div>
```
[the remark this answers is not in the document]
Comment (shubh0779) [reply SubmittedAssignments/Shubham_Shinde/Preview.html:40]: Change it

File: SubmittedAssignments/Shubham_Shinde/Preview.html [hunk 2]
```diff
@@ -0,0 +1,92 @@
...
+			<div class="certificate-logo">
+				<img src="./Img/102-removebg-preview.png">
+			</div>
+
+			<!-- Content -->
+			<div class="initial-content">This Certificate is presented to</div>
+			<div class="candidate-name">Name Suname</div>
+		</div>
+
+		<div class="content-info">
```
[the remark this answers is not in the document]
Comment (shubh0779) [reply SubmittedAssignments/Shubham_Shinde/Preview.html]: Done !

File: SubmittedAssignments/Shubham_Shinde/Preview.html [hunk 3]
```diff
@@ -0,0 +1,92 @@
...
+
+	<!-- <link rel="stylesheet" type="text/css" href="./css/bootstrap.min.css"> -->
+</head>
+
+<body>
+	<!--Certificate Wrapper-->
+	<div class="certificate-wrapper">
+
+		<!--Certificate background-->
+		<!-- <div class="certificate-background"> -->
```
Comment (laxmikant-gaykhee) [inline SubmittedAssignments/Shubham_Shinde/Preview.html:19]: remove this comment  **class="certificate-background"**

**Review by laxmikant-gaykhee — CHANGES_REQUESTED** [review]
change file name **certificate1.css** to **style.css** in **css folder**

File: SubmittedAssignments/Shubham_Shinde/Preview.html (hunk 3 above)
Comment (shubh0779) [reply SubmittedAssignments/Shubham_Shinde/Preview.html]: done

**Review by laxmikant-gaykhee — APPROVED** [review]

**Pull request merged**